scons-toolfile.spec

```diff
@@ -1,5 +1,5 @@
 ### RPM external scons-toolfile 2.0
-Requires: scons
+Requires: py3-scons
```

Comment:
@davidlange6 , this will fail to find `TOOL_ROOT`. Please rename `scons-toolfile.spec`  to `py3-scons-toolfile.spec` so that  https://github.com/cms-sw/cmsdist/blob/IB/CMSSW_12_0_X/master/scram-tools-post.file#L4 can properly find the `TOOL_ROOT` . Note that the xml file generated inside the `py3-scons-toolfile.spec` can remain the same.